### DISCUSSION TITLE
How is the size of a module taken into account

### DISCUSSION THREAD

**Discussion by julikru** [pvlib/pvlib-python#1298] [Q&A]
Hi,
I'm using the rest api to retrieve the generated energy and therefore the area is indirectly covered in the peak power variable as area * module_efficiency. 
Looking into the PVSystem class I see that in the constructor you can define modules_per_string and strings_per_inverter and also module parameters. I am wondering how the area is taken into account here? Is it done indirectly by the number of modules, as each module has a known area?
Thanks in advance

**Comment by cwhanse** [comment]
The module parameters include DC power at Standard Conditions (1000 W/m2 irradiance) in units of W (DC), not in W/m2 (or W/cm2), so the module area isn't needed.
